assets/src/scripts/components/hydrograph/time-series-graph.js

```diff
@@ -86,6 +88,63 @@ const plotAllMedianPoints = function (elem, {visible, xscale, yscale, seriesPoin
     });
 };
 
+/**
+ * Plots the Waterwatch flood level points for a multiple time series.
+ * @param  {Object} elem
+ * @param  {Function} xscale
+ * @param  {Function} yscale
+ * @param  {Number} modulo
+ * @param  {Array} points
+ */
+const plotFloodLevelPoints = function(elem, {xscale, yscale, points, classes}) {
+    const stepFunction = d3Line()
+        .curve(curveStepAfter)
+        .x(function (d) {
```

Comment:
So rather than using xscale to translate the date, you draw two points for each line. The y point would be the same, but the x point would just be the extent of the xscale (which you get with xscale.range()). I also believe that you don't need curveStepAfter for a two point line.

Reply:
I went ahead and used the y value and the value at each end of xscale.domain(), I also removed curveStepAfter.